scripts/release/publish-commands/publish-to-npm.js

```diff
@@ -30,6 +30,10 @@ const push = async ({cwd, dry, version}) => {
       const packageVersion = packageJSON.version;
 
       if (!dry) {
+        // Wait a couple of seconds before querying NPM for status;
+        // Anecdotally, querying too soon can result in a false negative.
+        await new Promise(resolve => setTimeout(resolve, 5000));
```

Comment:
This needs to get into standard library already....

Reply:
Yah